From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Anton Tayanovskyy <anton@pulumi.com>
Date: Mon, 2 Jun 2025 15:38:32 -0400
Subject: [PATCH] Re-export NewProvider for linking


diff --git a/shim/shim.go b/shim/shim.go
new file mode 100644
index 0000000000..c476df8cec
--- /dev/null
+++ b/shim/shim.go
@@ -0,0 +1,10 @@
+package shim
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/oracle/terraform-provider-oci/internal/provider"
+)
+
+func NewProvider() *schema.Provider {
+	return provider.Provider()
+}
